drud/ddev
# adds support for https in ddev router

## The Problem/Issue/Bug:
#232 OP - We want to support both HTTP and HTTPS for local dev without redirect.

## How this PR Solves The Problem:
- Updates to use provisional ssl tag for ddev-router, introducing https support w/ self-signed certs
- Updates site docker-compose.yml template to define new HTTPS_EXPOSE
- Updates router setup to ensure ports defined in HTTPS_EXPOSE are exposed on ddev-router.
- Updates successful start message to include both HTTP and HTTPS URL options (I'm not sure if this is what we want or if we want to do something different)

## Manual Testing Instructions:
This can be tested by building a binary from this PR and spinning up a test ddev site. You'll want to ensure you get a freshly-generated docker-compose for the site. The site should spin up successfully, and you should be able to access the site at both HTTP and HTTPS

## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:
